crates/collab_ui/src/collab_titlebar_item.rs

```diff
@@ -271,8 +276,11 @@ impl Render for CollabTitlebarItem {
                             .selected_style(ButtonStyle::Tinted(TintColor::Negative))
                             .icon_size(IconSize::Small)
                             .selected(is_deafened)
+                            .disabled(cfg!(not(target_os = "linux")))
```

Comment:
Hey @mikayla-maki, shouldn't this be `not(target_os = "macos")`? On current nightly these icons are disabled, so